bulma-compo/src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import './Navbar.css'
+import 'bulma/css/bulma.css';
+import CoolButton from './../coolbutton/CoolButton'
+
+const Navbar = props => {
+
+    return(
+        <nav class="navbar is-transparent">
+            <div class="navbar-brand">
+                <a class="navbar-item" href="https://bulma.io">
```

Comment:
```suggestion
        <nav className="navbar is-transparent">
            <div className="navbar-brand">
                <a className="navbar-item" href="https://bulma.io">
```
Para especificar una clase CSS, usa el atributo className.